app/subscriber/src/features/my-reports/utils/calcNextSend.ts

```diff
@@ -17,17 +17,15 @@ interface ISchedule {
 }
 
 function getNextReportSend(report: IReportModel): string {
-  // Set constants with the last sent info. If it's null consider the actual date
-  const sentOn: string | Date | undefined = report.instances.length
-    ? report.instances[0].sentOn
-    : undefined;
-
-  const lastSent: Date = sentOn === undefined ? new Date() : new Date(sentOn);
-
-  const lastWeekDay = lastSent.toLocaleDateString('en-US', { weekday: 'long' });
-
-  const lastTimeSent = lastSent.toLocaleTimeString('en-US', {
-    timeZone: 'UTC',
+  const currentTime: Date = new Date();
+  const currentWeekDay = currentTime.toLocaleDateString('en-US', {
+    timeZone: 'America/Vancouver',
+    weekday: 'long',
+  });
+  const currentTimeOfDay = currentTime.toLocaleTimeString('en-US', {
+    // TODO: Store User's preferred timezone in profile/settings to use for coverting/storing
+    // Server currently compares time in server local time, which is PST. Hence, use PST.
+    timeZone: 'America/Vancouver',
```

Comment:
Need a sanity check... I don't think last sent date is even relevant? You can't have a future send be in anytime before the present time... so we can just use the current time for calculating next send. @Fosol

Reply:
It depends... The last sent date does indicate whether it was sent.  However, I think your logic is sound.  Whether it was sent or not, the next calculated send date would be in the future anyway.